packages/web-app-files/src/views/PublicFiles.vue

```diff
@@ -124,17 +123,18 @@ export default {
   mixins: [
     MixinAccessibleBreadcrumb,
     MixinFileActions,
-    MixinFilesListPositioning,
     MixinFilesListPagination,
     MixinMountSideBar
   ],
 
   setup() {
-    useMutationObserver(['Files/UPSERT_RESOURCE'], ({ payload }) =>
-      nextTick(() => {
-        fileList.accentuateItem(payload.id)
-      })
-    )
+    const { refresh: refreshFileListHeaderPosition, y: fileListHeaderY } =
+      useFileListHeaderPosition()
+
+    useMutationSubscription(['Files/UPSERT_RESOURCE'], async (mutation) => {
```

Comment:
What's the reason for switching from exploded object syntax to an object var?

Reply:
we now return the whole mutation instead of just the payload like before, this way the type can be used to switch between actions.

https://vuex.vuejs.org/api/#subscribe

Reply:
yeah understood it that way, but instead of `(mutation)` you could write `({ payload, type })`, i.e. exploded object syntax. I find that easier to read. Is there a reason against it?

Reply:
Done in last commit